shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -222,6 +279,44 @@ private GetWorkerReportOptions getOptions(CommandLine cl) throws IOException {
     return workerOptions;
   }
 
+  /**
+   * Gets the formatted tier values of a worker.
+   *
+   * @param map the map to get worker tier values from
+   * @param workerName name of the worker
+   * @return the formatted tier values of the input worker name
+   */
+  private static String getWorkerFormattedTierValues(Map<String, Map<String, String>> map,
+      String workerName) {
+    return map.values().stream().map((tierMap)
+        -> (String.format("%-14s", tierMap.getOrDefault(workerName, "-"))))
+        .collect(Collectors.joining(""));
+  }
+
+  /**
+   * Initializes member variables used to collect worker info.
+   */
+  private void initVariables() {
+    mSumCapacityBytes = 0;
+    mSumUsedBytes = 0;
+    mSumCapacityBytesOnTierMap = new TreeMap<>(new TierNameComparator());
```

Comment:
You can do this:
```java
mSumCapacityBytesOnTierMap = new TreeMap<>(FileSystemAdminShellUtils::compareTierNames);
```

Reply:
Or 
```
mSumCapacityBytesOnTierMap = new TreeMap<>(Comparator.comparing(FileSystemAdminShellUtils::getTierRankValue));
```
So you can further remove `compareTierNames`

Reply:
Changed to 
```java
new TreeMap<>(FileSystemAdminShellUtils::compareTierNames)
```
Much clear and easy to use!
getTierRankValue gives all the tiers except MEM, SSD and HDD same value, which is not good for test.
Thanks for the good idea, I will try to make use of this new feature.!